src/scripts/articles/articleHTML.js

```diff
@@ -1,6 +1,7 @@
 //J.Kaset HTML representation of one article. 
 
 export const Article = (articleObj) => {
+  if(articleObj.userId === parseInt(sessionStorage.getItem("activeUser"))) {
```

Comment:
"activeUser does not need to be parsed into an integer because the id it is being compared to in the API is a string instead of a number."